docker-compose.e2e.yaml

```diff
@@ -26,25 +26,29 @@ services:
         --account 0x011967d88c6b79116bb879d4c2bc2c3caa23569edd85dfe0bc596846837bbc8e,0x56bc75e2d63100000
 
   contract_deployer:
-    image: pointnetwork/pointnetwork_deployer:dev
     container_name: deployer
+    build:
```

Comment:
@tankakatan building the image from the compose service definition is ok going forward or should we deploy a ready made image to DockerHub?

Reply:
We should deploy it. Building images in compose files is fine during development. But the e2e compose is used not only by Point contract devs, but also by other teams. It would be much more convenient for them to download the image rather than build it from scratch.